lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -153,15 +155,15 @@ WebProcessingServiceCatalogItem.prototype._load = function() {
                     }));
                 } else if (defined(outputs[i].Data.ComplexData)) {
                     var content = outputs[i].Data.ComplexData;
-                    if (outputs[i].Data.ComplexData.mimeType === 'text/csv' || outputs[i].Data.ComplexData.mimeType === 'application/json')
-                    {
-                        var colors = '';
-                        if (defined(outputs[i].colors)) {
-                            colors = 'colors="' + outputs[i].colors + '"';
-                        }
+                    if (outputs[i].Data.ComplexData.mimeType === 'text/csv') {
                         content = '<collapsible title="' + outputs[i].Title + '" open="' + (i === 0 ? 'true' : 'false') + '">';
-                        content += '<chart ' + colors + ' can-download="true" hide-buttons="false" title="' + outputs[i].Title + '" data=\'' + outputs[i].Data.ComplexData + '\' styling="histogram"></chart>';
+                        content += '<chart can-download="true" hide-buttons="false" title="' + outputs[i].Title + '" data=\'' + outputs[i].Data.ComplexData + '\' styling="histogram"></chart>';
                         content += '</collapsible>';
+                    } else if (outputs[i].Data.ComplexData.mimeType === 'application/json') {
```

Comment:
I'm not sure it's fair to assume that ComplexData with mime type `application/json` is always a TerriaJS init file.  Let's instead use `application/vnd.terriajs.init+json`.  In theory we can officially register that (http://www.iana.org/form/media-types) but it might be overkill.